charts/ocis/values.yaml

```diff
@@ -263,6 +263,17 @@ securityContext:
   # -- Group ID that all processes within any containers will run with.
   runAsGroup: 1000
 
+# -- TopologySpreadConstraints controls how Pods of a service are spread across the cluster among failure-domains.
+# Defaults to allow skew no more then 1 node per node.
+# Passed through tpl and therefore needs to be configured as string.
+topologySpreadConstraints: |
+  - maxSkew: 1
+    topologyKey: kubernetes.io/hostname
+    whenUnsatisfiable: DoNotSchedule
+    labelSelector:
+      matchLabels:
+        app: {{ .appName }}
```

Comment:
I think we shouldn't set any default topologySpreadConstraints for now. Maybe just add a commented example.

Defining useful constraints requires some knowledge about the cluster setup, we don't really know where this is actually deployed.

BTW, shouldn't we also allow to set affinity rules? My feeling is that people are more familar with those  and in many case they should be "good enough". Also `affinity` and `topologySpreadConstraints` don't exclude each other. (Might be a topic for another PR)